.travis.yml

```diff
@@ -17,4 +17,13 @@ git:
 #  - julia -e 'Pkg.clone(pwd()); Pkg.build("StatsBase"); Pkg.test("StatsBase"; coverage=true)'
 after_success:
   - julia -e 'using Pkg; Pkg.add("Coverage"); using Coverage; Coveralls.submit(process_folder()); Codecov.submit(process_folder())';
-  - julia -e 'using Pkg; Pkg.add("Documenter"); include(joinpath("docs", "make.jl"))'
+jobs:
+  include:
+    - stage: "Documentation"
+      julia: 1.0
+      os: linux
+      script:
+        - julia --project=docs/ -e 'using Pkg; Pkg.instantiate();
+                                    Pkg.develop(PackageSpec(path=pwd()))'
```

Comment:
Develop should come before instantiate

Reply:
Fixed. I just copied this from DataFrames. Any idea whether it really makes a difference?

Reply:
Yes, if you instantiate first, some weird version resolution things can happen. I've run into them before and asked in #pkg-usage on Slack, and this was Fredrik's fix. I can't recall the full details of the problems it can cause though.